remark-custom-rules/title-capitalization/remarkLintTitleCapitalization.js

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
In the project we use TS as much as possible.

But if you want to publish plugin as open source project, it is easier to publish JS projects than TS.